src/brackets.js

```diff
@@ -437,7 +438,9 @@ define(function (require, exports, module) {
         }
 
         if (brackets.platform === "win" && !brackets.inBrowser) {
-            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true).on("change", enableOrDisableWinScrollFix);
+            PreferencesManager.definePreference("_windowsScrollFix", "boolean", true, {
```

Comment:
Is there some way to exclude a preference?
This is more or less a "hidden" preference that no one should ever need to change himself (and it will also go away at some point), so I'd appreciate it if this pref wasn't shown.
It wouldn't be much of an issue normally, but right now, this pref is the first to show up in the list...